packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -55,69 +56,76 @@ function useMongoNavData({
   const [data, setData] = useState<DataInterface | undefined>(undefined);
 
   const endpointURI = `${hosts.cloud}/user/shared`;
+  const body: PostBodyInterface | null = useObjectDependency(
```

Comment:
>I'm confused by all the refactoring in here. What changed aside from the code itself? Is it functionally the same? What issues were there in the previous code?

The issues with the previous code could be summarized as the `useEffect` depending on a bunch of things and not declaring them or not properly declaring them in the deps array ("properly" defined as in a way that the linter can detect). The fixes were:
- Move definitions into the effect if they weren't needed elsewhere
- Make derived objects use `useObjectDependency` to avoid manually having to specify the deps array of `useMemo`s.
- Simplify the code to reduce number of dependencies needed (and used it as an excuse to use AbortController)


>Can we do a small sniff for browser support before using it?

We use the Fetch API for these requests which is also not supported by IE, so I'm not sure how things could be working now. It doesn't look like we're polyfilling it.

Reply:
So mostly the use of `useObjectDependency` resulted in these changes. Alrighty.

Let's not worry about Fetch for now...didn't realize that either.